### PR TITLE
CC30 and CC32

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

**Comment by Gr8-Dayne** [comment]
Code Challenge 30 - Hash Table Implementation

 - [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

**Comment by Gr8-Dayne** [comment]
I forgot to change branches for CC32, I'll submit both with this one, with clear labels where needed.

**Comment by Gr8-Dayne** [comment]
Code Challenge 32 - Repeated Word 

- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: challenges/src/main/java/hashtable/Hashtable.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package hashtable;
+
+
+public class Hashtable {
+
+    int size;
+    String[] map;
```
Comment (JamesDansie) [inline challenges/src/main/java/hashtable/Hashtable.java]: Should be array of linked lists of a "storage" class. So something like;
List<Storage>[] map;

Where Storage has a key/value pair.
